### PR TITLE
Add upload_and_queue method for new uploader testing

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

File: scripts/upload.py [hunk 1]
```diff
@@ -12,6 +13,35 @@ def get_unique_name(filename, unique_id) -> str:
...
+        files = glob(globpath, recursive=True)
+
+        for infile in files:
+            blob_name = get_unique_name(infile, os.getenv('HELIX_WORKITEM_ID'))
+
+            getLogger().info("uploading {} to storage account".format(infile))
+
+            blob_client = BlobClient(blob_url=storage_account_uri, container=container, blob=blob_name, credential=sas_token)
+            
+            with open(infile, "rb") as data:
```
Comment (billwert) [inline scripts/upload.py]: can we call `upload` below here instead of duplicating the code?

Comment (DrewScoggins) [reply scripts/upload.py]: Yeah, seems great. Not sure why I didn't do this in the first place.

**Comment by DrewScoggins** [comment]
I ended up having to make a few more changes than I wanted. Want to take another look @billwert?

File: eng/performance/benchmark_jobs.yml [hunk 2]
```diff
@@ -73,10 +73,10 @@ jobs:
             value: ''
           - ${{ if eq(parameters.osName, 'windows') }}:
             - name: HelixPreCommand
-              value: 'py -3 -m pip install --user --pre --upgrade azure-storage-blob;set "PERFLAB_UPLOAD_TOKEN=$(PerfCommandUploadToken)";$(AdditionalHelixPreCommands)'
+              value: 'py -3 -m pip install --user --pre --upgrade azure-storage-blob;py -3 -m pip install --user --pre --upgrade azure-storage-queue;set "PERFLAB_UPLOAD_TOKEN=$(PerfCommandUploadToken)";$(AdditionalHelixPreCommands)'
```
[the remark this answers is not in the document]
Comment (DrewScoggins) [reply eng/performance/benchmark_jobs.yml]: Indeed. I'l make this happen.

File: scripts/benchmarks_ci.py [hunk 3]
```diff
@@ -252,7 +252,10 @@ def __main(args: list) -> int:
                 '**',
                 '*perf-lab-report.json')
 
-            upload.upload(globpath, UPLOAD_CONTAINER, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
+            #No queue insertion
+            upload.upload(globpath, UPLOAD_CONTAINER, None, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
+            #With queue insertion
+            upload.upload(globpath, 'resultsandbox', UPLOAD_QUEUE, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
```
[the remark this answers is not in the document]
Comment (DrewScoggins) [reply scripts/benchmarks_ci.py]: Yup!

**Pull request merged**